Users/Install.rst

```diff
@@ -299,7 +299,8 @@ Installation Errors
 In case you are getting
 ``ValueError:('Expected version spec in', 'appdirs ~=1.4.0', 'at',
 ' ~=1.4.0')``, then don't panic. It happens when you are using an outdated
-version of pip that doesn't support our version specifiers yet.
+version of pip that doesn't support our `version specifiers
+<https://www.python.org/dev/peps/pep-0440/#version-specifiers>`_ yet.
 
 
     Ideally, you have to create a virtual environment with a newer pip:
```

Comment:
For me, the version specifiers link renders fine.
![vs](https://user-images.githubusercontent.com/206581/27053372-63c781c6-4fbd-11e7-837e-31e47e0f6c79.png)